src/test/java/e2e/pages/AddNewPhoneNumberDialog.java

```diff
@@ -0,0 +1,54 @@
+package e2e.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+
+public class AddNewPhoneNumberDialog extends Header{
```

Comment:
Why here you inherit from Header?
Need to inherit from BasePage

Reply:
Fixed